core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3643,30 +3655,30 @@ private static String javadocLink(String fullyQualifiedClassname) {
         "alluxio.underfs.oss.connection.timeout";
     public static final String UNDERFS_OSS_CONNECT_TTL = "alluxio.underfs.oss.connection.ttl";
     public static final String UNDERFS_OSS_SOCKET_TIMEOUT = "alluxio.underfs.oss.socket.timeout";
-    public static final String UNDERFS_S3A_BULK_DELETE_ENABLED =
-        "alluxio.underfs.s3a.bulk.delete.enabled";
-    public static final String UNDERFS_S3A_DEFAULT_MODE = "alluxio.underfs.s3a.default.mode";
-    public static final String UNDERFS_S3A_DIRECTORY_SUFFIX =
-        "alluxio.underfs.s3a.directory.suffix";
-    public static final String UNDERFS_S3A_INHERIT_ACL = "alluxio.underfs.s3a.inherit_acl";
-    public static final String UNDERFS_S3A_INTERMEDIATE_UPLOAD_CLEAN_AGE =
-        "alluxio.underfs.s3a.intermediate.upload.clean.age";
-    public static final String UNDERFS_S3A_LIST_OBJECTS_VERSION_1 =
-        "alluxio.underfs.s3a.list.objects.v1";
-    public static final String UNDERFS_S3A_REQUEST_TIMEOUT_MS =
-        "alluxio.underfs.s3a.request.timeout";
-    public static final String UNDERFS_S3A_SECURE_HTTP_ENABLED =
-        "alluxio.underfs.s3a.secure.http.enabled";
-    public static final String UNDERFS_S3A_SERVER_SIDE_ENCRYPTION_ENABLED =
-        "alluxio.underfs.s3a.server.side.encryption.enabled";
-    public static final String UNDERFS_S3A_SIGNER_ALGORITHM =
-        "alluxio.underfs.s3a.signer.algorithm";
-    public static final String UNDERFS_S3A_SOCKET_TIMEOUT_MS =
-        "alluxio.underfs.s3a.socket.timeout";
-    public static final String UNDERFS_S3A_STREAMING_UPLOAD_ENABLED =
-        "alluxio.underfs.s3a.streaming.upload.enabled";
-    public static final String UNDERFS_S3A_STREAMING_UPLOAD_PARTITION_SIZE =
-        "alluxio.underfs.s3a.streaming.upload.partition.size";
+    public static final String UNDERFS_S3_BULK_DELETE_ENABLED =
+        "alluxio.underfs.s3.bulk.delete.enabled";
+    public static final String UNDERFS_S3_DEFAULT_MODE = "alluxio.underfs.s3.default.mode";
+    public static final String UNDERFS_S3_DIRECTORY_SUFFIX =
+        "alluxio.underfs.s3.directory.suffix";
+    public static final String UNDERFS_S3_INHERIT_ACL = "alluxio.underfs.s3.inherit.acl";
```

Comment:
@calvinjia I changed this property to `alluxio.underfs.s3.inherit.acl` to be consistent with other property names. Let me know if you are good with it